target/linux/realtek/dts/rtl9312_plasmacloud_psx28.dts

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+/dts-v1/;
+
+#include "rtl9312_plasmacloud_common.dtsi"
+
+/ {
+	compatible = "plasmacloud,psx28", "realtek,rtl838x-soc";
```

Comment:
Please align to upstream and use rtl9312-soc. See https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/arch/mips/boot/dts/realtek?h=v6.17